common/lib/core-interfaces/package.json

```diff
@@ -77,6 +77,22 @@
         "RemovedInterfaceDeclaration_IProvideFluidConfiguration": {
           "backCompat": false,
           "forwardCompat": false
+        },
```

Comment:
The version of this package needs to be updated. as right now it is the release version, it should be bumped to the next version, @heliocliu

Reply:
working on it :d

Reply:
in terms of process, we probably want bump next immediately after we merge to main. My main worry is pre-release packages coming from the different branches as people make and consume changes across the repo

Reply:
that is how things are written in the instructions, but that didn't happen this time due to unexpected stuff with versions that was messing up the tooling plus some test failures that were messing up builds for the prerelease pipelines 🤕